aiohttp/web_fileresponse.py

```diff
@@ -334,8 +337,8 @@ def __init__(self, path: Union[str, pathlib.Path],
             self.headers[hdrs.CONTENT_RANGE] = 'bytes {0}-{1}/{2}'.format(
                 real_start, real_start + count - 1, file_size)
 
-        with filepath.open('rb') as fobj:
+        with (await loop.run_in_executor(None, filepath.open, 'rb')) as fobj:
```

Comment:
@Tolmachofof blocking close of the fobj

Reply:
@aamalev thanks for your comment!
I think that something like ```_FileContextManager``` would solve this problem. 
Simplified example:

```python
class _FileContextManager:

    def __init__(self, filepath, flags, start):
        self._filepath = filepath
        self._flags = flags
        self._start = start
        self._fobj = None

    async def __aenter__(self):
        loop = asyncio.get_event_loop()
        self._fobj = loop.run_in_executor(
            None, self._open, self._filepath, self._start
        )
        return self._fobj

    def _open(self, filepath, start):
        fobj = filepath.open(self._flags)
        if start:
            fobj.seek(start)
        return fobj

    async def __aexit__(self, exc_type, exc_val, exc_tb):
        loop = asyncio.get_event_loop()
        await loop.run_in_executor(None, self._fobj.close)
        self._fobj = None
```
If this acceptable i can make a PR

Reply:
I think the conversion `with` to `try..finally` is more suitable. `_FileContextManager` need members loop and executor, and this is too much for a one-time code

Reply:
You are right. ```try..finally``` statement sounds good. I will make a PR.